.github/workflows/hardhat-tests.yml

```diff
@@ -55,9 +42,6 @@ jobs:
             hardhat-tests/test/internal/hardhat-network/stack-traces/test-files/artifacts
           key: hardhat-network-stack-traces-tests-${{ hashFiles('hardhat-tests/test/internal/hardhat-network/stack-traces/test-files/**/*.sol') }}-${{ hashFiles('hardhat-tests/test/internal/hardhat-network/stack-traces/test-files/**/test.json') }}-${{ hashFiles('hardhat-tests/test/internal/hardhat-network/stack-traces/**/*.ts') }}
 
-      - name: Build
```

Comment:
Just validating my understanding: this is not necessary because the `test` command has a pre-test step that builds.

Reply:
Yes, exactly. That wasn't the case before (because the `pretest` script was only running `build:edr`), but now both EDR is built and the tests code is type-checked before running the tests.